### ISSUE TITLE
Edit Scheduled Transaction inconsistent date formats

### ISSUE THREAD

**Issue by luisdev** [moneymanagerex/android-money-manager-ex#1739]
**Describe the bug**
The **Due Date** and **Payment Date** date picker objects on the Scheduled Transaction screen show the dates in different date formats.

**To Reproduce**
1. Open the 'Scheduled Transactions' screen - the problematic behavior is the same whether you edit an existing Scheduled Transaction or if you create a new one.
2. Look at how the dates are displayed in the _Due Date_ and _Payment Date_ fields. _Due Date_ is displayed in the short format, **Mon 2024-07-22**, while the _Payment Date_ is displayed in the long format, **Monday, 22 July 2024**

**Expected behavior**
The dates on both the date picker controls on the 'Scheduled Transactions' screen should be displayed in the same format.

**Screenshots**
![Screenshot_20240722_114949_Money Manager Ex.jpg](https://github.com/user-attachments/assets/20fffe96-32d9-4b3e-a15b-853f1424060d)

**Device Information:**
 - Device: Samsung Galaxy S22 Ultra
 - OS: Android 14; One UI 6.1
 - Storage: Phone storage + Syncthing
 - App Version: 2024.07.04 (1035)

**Additional context**
N/A

**Comment by wolfsolver** [comment]
Well noted. I use this always and I didn't see.
I will fix in a short :)

**Comment by wolfsolver** [comment]
Fixed in my PR (payementDate take care of Date Format in setting), but did not undertand why is not completly allingned from a UI point of view.

![image](https://github.com/user-attachments/assets/8dd59fa5-ddd4-4801-b23b-2f14a145cf44)

**Comment by luisdev** [comment]
> Fixed in my PR (payementDate take care of Date Format in setting), but did not undertand why is not completly allingned from a UI point of view.
> 
> ![image](https://private-user-images.githubusercontent.com/3205448/350977407-8dd59fa5-ddd4-4801-b23b-2f14a145cf44.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE5ODQyNjcsIm5iZiI6MTcyMTk4Mzk2NywicGF0aCI6Ii8zMjA1NDQ4LzM1MDk3NzQwNy04ZGQ1OWZhNS1kZGQ0LTQ4MDEtYjIzYi0yZjE0YTE0NWNmNDQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDcyNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA3MjZUMDg1MjQ3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZTAyMmUzOTliNTI0YmM2Njc5ZGE0NzAyMmRkMzIwNWI0ZTlkN2M0ZjYxYjhmODlkZjkzNGQ5YTM1NTJiN2QzOSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.M0voPlGQUACaEs1A6b8krcHekEXJn7WJauoxKWe_M68)

Thanks. Where is that date format set? Does AMMEX get the format from the Android system settings?

**Comment by wolfsolver** [comment]
Date format is under ammex setting -> database

**Comment by luisdev** [comment]
I checked version 2024.08.01.1036 and this issue seems to be fixed! Thank you.

I did notice that the Left / Right arrows in the Due Date field are bigger and thinner than the Left / Right arrows in the Payment Date field.

![ammex](https://github.com/user-attachments/assets/6d5d01e9-68ef-422c-a240-3e8e4739b58c)

Could that be why the dates in the two fields are not horizontally aligned, like you stated?
